### ISSUE TITLE
Case sensitive property names show wrong value in documentation

### ISSUE THREAD

**Issue by incutonez** [senchalabs/jsduck#597]
Doing some documentation in my Node.js app, and I decided to use JSDuck for parsing it... I realize it's not meant for Node, but it does a tremendous job at creating documentation, so I figured I'd give it a try.

Anyway, I've got this sample code:

```
/**
  * @class Blah
  * @author Me
  * @docauthor Me
 */
modules.export = function Blah() {
  /** @property */
  var SOMETHING = 'SOMETHING';
  /** @property */
  var Something = 'Something';
  /** @property */
  var something = 'something';
};
```

I then go to my index.html of the documentation that's generated and see that all worked well, except for the values of SOMETHING and Something... they appear as Object, instead of the string value.  something (all lower case) shows the correct value.  Here's an image of what I'm talking about:

![bugorfeature](https://cloud.githubusercontent.com/assets/1584135/5650339/b124f83c-965a-11e4-8fcb-07f7a54ff3e6.png)

Apologies if this has been reported already, but I couldn't find anything relevant in the search.  My JSDuck version is JSDuck 5.3.4, using Ruby 2.1.1.

**Comment by nene** [comment]
Thanks for reporting. It's a bug.

I guess JSDuck first thinks Something and SOMETHING are going to be a classes as they begin with uppercase letter. Later it turns out, that they're explicitly documented as as @properties but at that point the information of them being a string is already lost.

Until this is fixed, you can just explicitly document all these as strings:

```
module.exports = function Blah() {
  /** @property {String} */
  var SOMETHING = 'SOMETHING';
  /** @property {String} */
  var Something = 'Something';
  /** @property {String} */
  var something = 'something';
};
```

**Comment by incutonez** [comment]
Yeah that definitely makes sense for Something.  Documenting them as String does indeed make them appear as such in the docs, but it doesn't show their default value... I'd have to explicitly set it after {String}... which seems a bit redundant, as I'd have to maintain a comment of the constant's value.  Sorry if that sounds a bit nitpicky.
